.github/workflows/handler-test.yml

```diff
@@ -1,5 +1,9 @@
 name: Pull Request Test Handler
 
+# This is used to define anchors for repeated keys or configuration blocks.
+tfe_parameters:
+  working-directory: &PUBLIC_WORK_DIR ./tests/public-install
```

Comment:
GitHub Actions has a way to set the working directory by default which may be preferable to using YAML anchors: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#defaultsrun

Reply:
I don't disagree with using GitHub Action defaults, we use them for the `run.shell` value, but the reason for using YAML anchors is because we'll be leveraging different working directories for `k6`, `private-install` and `private-tcp-install` in the future steps for this handler.

Reply:
😭 
```
The workflow is not valid. .github/workflows/handler-test.yml (Line: 4, Col: 1): Unexpected value 'tfe_parameters' .github/workflows/handler-test.yml: Anchors are not currently supported. Remove the anchor 'PUBLIC_WORK_DIR'
```

I'll refactor the anchors to be variables or something similar in the action so they can work the same way.